### PR TITLE
String error fix

### PR DESCRIPTION
**Pull request by Venkat19967**
**Description of the change:**
The change has fmt.Sprintf("...") removed from sb.WriteString(fmt.Sprintf("...")) and two other variable assignment in 3 files at various places.

This is because the latest version throws error stating that fmt.Sprintf() is doing an extra step of converting a string to a string. 

I have also done formatting of code in one of the section due to this error:
internal/plugins/golang/plugin.go:46: File is not `goimports`-ed (goimports)

after pretty print I am able to make the sdk.

**Motivation for the change:**

I have installed the latest version of Go and a couple of features are throwing errors making it difficult to build.

### CODE REVIEW

File: internal/plugins/golang/plugin.go [hunk 1]
```diff
@@ -43,9 +43,11 @@ var (
 
 type Plugin struct{}
 
-func (Plugin) Name() string                       { return (kbgov2.Plugin{}).Name() }
-func (Plugin) Version() plugin.Version            { return (kbgov2.Plugin{}).Version() }
-func (Plugin) SupportedProjectVersions() []string { return (kbgov2.Plugin{}).SupportedProjectVersions() }
+func (Plugin) Name() string            { return (kbgov2.Plugin{}).Name() }
```
Comment (estroz) [inline internal/plugins/golang/plugin.go:46]: Revert this change please.

**Comment by camilamacedo86** [comment]
HI @Venkat19967,

The error:

```
I have also done formatting of code in one of the section due to this error:
internal/plugins/golang/plugin.go:46: File is not goimports-ed (goimports)
```

It is regards to spaces. You can run `make lint-fix` to fix the  `goimports-ed` issues.
